src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -75,12 +75,23 @@ For [ping monitors](#create-ping), these include:
 
 For [simple browser monitors](#create-simple-browser):
 
-* <var>BROWSER_TYPE</var>: The browser used by your monitor. "CHROME" is the only accepted value.
-* <var>BROWSER_VERSION</var>: The browser version used by your monitor. "100" is the only accepted value. 
+* <var>RUNTIME_TYPE</var>: The runtime type used by your monitor. "CHROME_BROWSER" is the only accepted value.
+* <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. "100" is the only accepted value. 
+* <var>SCRIPT_LANGUAGE</var>: The language used in your monitor. "JAVASCRIPT" is the only accepted value. 
 
-For [scripted API monitors](#scripted-API):
+For [scripted API monitors](#create-scripted-API):
 
 * <var>SCRIPT_CONTENT</var>: The actual contents of the script. This should **not** be based64 encoded. 
+* <var>RUNTIME_TYPE</var>: The runtime type used by your monitor. "NODE_API" is the only accepted value.
+* <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. "16.10.0" is the only accepted value. 
```

Comment:
```suggestion
* <var>RUNTIME_TYPE_VERSION</var>: The runtime type version used by your monitor. The only accepted value is `16.10.0`. 
```

Reply:
(We try not to start sentences with numbers for formatting/style reasons. I added a codeblock around the value so users know not to include the terminal punctuation.)